.travis.yml

```diff
@@ -2,7 +2,7 @@ language: rust
 rust:
   - stable
   - beta
-  - 1.22.0
+  - 1.24.0
```

Comment:
What change required this?

Reply:
rustfmt only supported on 1.24.0 and beyond :/

Reply:
Right, but we can just...not format on travis with 1.22? We are only going to require formatting against stable/1.34 or whatever, anyway, no?

Reply:
Ah right, rustfmt doesn't need to support 1.22 for code that uses it to build on 1.22 :woman_facepalming: fixed!